rag_text2sql/rag_text_to_sql_ui.py

```
@@ -0,0 +1,27 @@
+import streamlit as st
+import nest_asyncio
```

Ruff (0.8.2)

`nest_asyncio` imported but unused

Remove unused import: `nest_asyncio`

(F401)